packages/functional-tests/tests/oauth/oauthPromptNone.spec.ts

```diff
@@ -5,7 +5,8 @@
 import { TestAccountTracker } from '../../lib/testAccountTracker';
 import { Page, expect, test } from '../../lib/fixtures/standard';
 import { BaseTarget, Credentials } from '../../lib/targets/base';
-import { LoginPage } from '../../pages/login';
+import { SigninPage } from '../../pages/signin';
+import { SettingsPage } from '../../pages/settings';
```

Comment:
Might be worth doing an [organize imports cmd](https://code.visualstudio.com/docs/languages/javascript#_organize-imports) on these files

```suggestion
import { SettingsPage } from '../../pages/settings';
import { SigninPage } from '../../pages/signin';
```

Reply:
Bigger diff coming right up ;)